test/README.md

```diff
@@ -23,3 +23,104 @@ Some examples of test harnesses using these tests, from the source repository:
 
 For users of Visual Studio Code, a [settings file](./.vscode/settings.json) is included that enables schema validation while editing the test files.
+
+## Test Functions
+
+As the behaviour of some of the default registry functions
+such as `:number` and `:datetime`
+is dependent on locale-specific data and may vary between implementations,
+the following functions are defined for **test use only**:
+
+### `:test:function`
+
+This function is valid both as a selector and as a formatter.
+
+#### Operands
+
+The function `:test:function` requires a [Number Operand](/spec/registry.md#number-operands) as its _operand_.
+
+#### Options
+
+The only option `:test:function` recognizes is `fd`,
+a [digit size option](/spec/registry.md) for which only `0` and `1` are valid values.
+
+All other options and their values are ignored.
+
+#### Behavior
+
+When resolving a `:test:function` expression,
+its _Input_ and _FD_ values are determined as follows:
+
+1. Let _FD_ be `0`.
+1. Let _arg_ be the resolved value of the expression operand.
+1. If _arg_ is the resolved value of an expression
+   with a `:test:function`, `:test:select`, or `:test:format` annotation
+   for which resolution has succeeded, then
+   1. Let _Input_ be the _Input_ value of _arg_.
+   1. Set _FD_ to be _FD_ value of _arg_.
+1. Else if _arg_ is a numerical value
+   or a string matching the `number-literal` production, then
+   1. Let _Input_ be the numerical value of _arg_.
+1. Else,
+   1. Emit "bad-input" Resolution Error.
+   1. Use a fallback representation as the resolved value of the expression.
+      Further steps of this algorithm are not followed.
+1. If the `fd` option is set, then
+   1. If its value resolves to a numerical integer value `0` or `1`
+      or their corresponding string representations `'0'` or `'1'`, then
+      1. Set _FD_ to be the numerical value of the option.
+   1. Else if its value is not an unresolved value set by [Option Resolution](/spec/formatting.md#option-resolution),
+      1. Emit "bad-option" Resolution Error.
+      1. Use a fallback representation as the resolved value of the expression.
+
+When `:test:function` is used as a selector,
+the behaviour of calling it as the `rv` value of MatchSelectorKeys(`rv`, `keys`)
+(see [Resolve Preferences](/spec/formatting.md#resolve-preferences) for more information)
+depends on its _Input_ and _FD_ values.
+
+- If the _Input_ is `1` and _FD_ is `1`,
+  the method will return some slice of the list « `'1.0'`, `'1'` »,
+  depending on whether those values are included in `keys`.
+- If the _Input_ is `1` and _FD_ is `0`,
+  the method will return the list « `'1'` » if `keys` includes `'1'`, or an empty list otherwise.
+- If the _Input_ is any other value, the method will return an empty list.
+
+When an expression with a `:test:function` annotation is assigned to a variable by a declaration
+and that variable is used as an option value,
+its resolved value is the _Input_ value.
```

Comment:
...line breaks and italics for terms...

Reply:
Introducing italicized terms here would make it more difficult to discern the local _Input_ reference, which I'd prefer to avoid. Can it be avoided, or is there some other styling I should Use for _Input_ and _FD_?

Reply:
Use `code` like `Input`, perhaps??